jumeg/connectivity/con_viz.py

```diff
@@ -547,16 +547,26 @@ def plot_grouped_connectivity_circle(yaml_fname, con, orig_labels,
     NOTE: yaml order fix helps preserves the order of entries in the yaml file.
     """
     import matplotlib.pyplot as plt
+
     # read the yaml file with grouping
     if op.isfile(yaml_fname):
         with open(yaml_fname, 'r') as f:
             label_groups = yaml.safe_load(f)
     else:
         print('%s - File not found.' % yaml_fname)
         sys.exit()
+    if yaml_color_fname is None:
```

Comment:
So this yaml file will contain a different grouping for colors only? I think this will be a bit too confusing to have two groupings. Do you have a working example for this?

Reply:
![Memory](https://user-images.githubusercontent.com/25769508/135410862-373b89f0-baf4-4e6e-8735-7bf0d2f9bf11.png)

Reply:
Okay looks good, you can add this sample yaml_color_fname file in jumeg/data and include this example.